### DISCUSSION TITLE
passenger details and payment

### DISCUSSION THREAD

**Discussion by skdss1632** [shivamguys/irctc-cypress-automation#50] [Ideas]
1. taking 25 second to fill two passeneger details, instead of filling passeneger details you can select from the master list it should save lot of time hardly it will take 5 seconds..
2. add new payment options i.e from irctc wallet it is much faster than any other for booking tatkal

**Comment by Vatsa-Vaibhav** [comment]
True

**Reply by shivamguys** [reply]
Irctc wallet is not a general use case for all , although its quick but cant be added as a general purpose
